### PR TITLE
Follow up to PS-5772 : 8.0.16 encryption reconciliation

### PR DESCRIPTION
**Pull request by satya-bodapati**
Fixes innodb.percona_force_encryption test with new variable
default_table_encryption and privilege changes

### CODE REVIEW

File: mysql-test/suite/innodb/t/percona_force_encryption.test [hunk 1]
```diff
@@ -8,19 +8,44 @@ call mtr.add_suppression("Function 'keyring_file' already exists");
...
+GRANT CREATE TABLESPACE, PROCESS, SYSTEM_VARIABLES_ADMIN, SUPER, RELOAD, TABLE_ENCRYPTION_ADMIN ON *.* TO u1@localhost;
+connect (con1, localhost, u1,, db1);
+
+let $initial_encrypt_tables_value = `select @@default_table_encryption`;
 
 create table t1_default_explicit_on (a int) engine=innodb encryption="Y";
 create table t2_default_explicit_off (a int) engine=innodb encryption="N";
 
-set global innodb_encrypt_tables='ON';
+set default_table_encryption='ON';
```
Comment (laurynas-biveinis) [inline mysql-test/suite/innodb/t/percona_force_encryption.test]: Consistent SQL keyword case please
```suggestion
SET default_table_encryption='ON';
```

File: mysql-test/suite/innodb/t/percona_force_encryption.test [hunk 2]
```diff
@@ -58,25 +86,50 @@ show create table t6_force_default;
...
+
 alter table t3_on_default;
---error ER_INVALID_ENCRYPTION_OPTION
 alter table t4_on_explicit_off encryption="N";
 
-set global innodb_encrypt_tables='ON';
---error ER_ALTER_OPERATION_NOT_SUPPORTED_REASON
+set global default_table_encryption='ON';
+
+--error  ER_ALTER_OPERATION_NOT_SUPPORTED_REASON
```
Comment (laurynas-biveinis) [inline mysql-test/suite/innodb/t/percona_force_encryption.test]:
```suggestion
--error ER_ALTER_OPERATION_NOT_SUPPORTED_REASON
```

File: mysql-test/suite/innodb/t/percona_force_encryption.test [hunk 3]
```diff
@@ -141,5 +204,9 @@ drop tablespace ts_encrypted1;
 drop tablespace ts_encrypted2;
 drop tablespace ts_unencrypted1;
 
-eval set global innodb_encrypt_tables=$initial_encrypt_tables_value;
-
+eval set global default_table_encryption=$initial_encrypt_tables_value;
```
Comment (laurynas-biveinis) [inline mysql-test/suite/innodb/t/percona_force_encryption.test]:
```suggestion
eval SET global default_table_encryption=$initial_encrypt_tables_value;
```

File: mysql-test/suite/innodb/t/percona_force_encryption.test [hunk 4]
```diff
@@ -8,19 +8,44 @@ call mtr.add_suppression("Function 'keyring_file' already exists");
...
+
+CREATE DATABASE db1;
+USE db1;
+
+CREATE USER u1@localhost;
+GRANT ALL ON db1.* TO u1@localhost;
+GRANT CREATE TABLESPACE, PROCESS, SYSTEM_VARIABLES_ADMIN, SUPER, RELOAD, TABLE_ENCRYPTION_ADMIN ON *.* TO u1@localhost;
+connect (con1, localhost, u1,, db1);
+
+let $initial_encrypt_tables_value = `select @@default_table_encryption`;
```
Comment (laurynas-biveinis) [inline mysql-test/suite/innodb/t/percona_force_encryption.test]: Nit: I'd rename to initial_default_table_encryption_value

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

**Comment by satya-bodapati** [comment]
ready for review

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**